docs/source/user-guide/configuration/use-condarc.rst

```diff
@@ -346,7 +346,7 @@ here overrides that default:
 
   proxy_servers:
       http: http://user:pass@corp.com:8080
-      https: https://user:pass@corp.com:8080
+      https: http://user:pass@corp.com:8080
```

Comment:
````suggestion
      https: https://user:pass@corp.com:8080
      
```{admonition} Mixing HTTPS and HTTP
For the ``https`` key, it is okay to use a server which only accepts the HTTP protocol (e.g. ``http://user:pass@corp.com:8080``
```
````
@padeoe,

My only concern with your suggested improvement is that it may also confuse readers who would expect the `https` key to point at an HTTPS server.

I have another suggestion though (please see above). We could add an admonition directly below the example the informs users about being able to use either HTTP or HTTPS for the `https` key.

Let me know what you think.

Reply:
Thank you for your prompt response.

I understand your concern about potentially confusing readers. However, I'd like to clarify a key point: the protocol of the proxy server (HTTP or HTTPS) is typically fixed and not interchangeable based on the type of traffic (http or https) it's routing.

For instance, if a user has an HTTP proxy server, both the http and https keys should point to an http:// URL. Similarly, if a user has an HTTPS proxy server, both keys should point to an https:// URL. It's not a mix-and-match scenario.

The current documentation might lead users to believe that they should use an https:// URL for the https key even if their proxy server only supports HTTP. This could result in misconfigurations.

To make this clearer, perhaps we could rephrase the admonition:
```{admonition} Note on Proxy Server Protocol
The protocol in the URL (either ``http://`` or ``https://``) should match the actual protocol of your proxy server. The keys ``http`` and ``https`` merely indicate the type of traffic to route, not the protocol of the proxy server itself. Ensure that both keys use the correct protocol based on your proxy server's configuration.
```
Feel free to adjust the wording as you see fit!

Reply:
@travishathaway

Reply:
@padeoe,

I think the admonition you propose makes sense. I will get the opinion of some colleagues on this too.